online-shopping-website/src/App.css

```diff
@@ -1,3 +1,40 @@
-.App-content {
-  padding: 0.5em;
-}
\ No newline at end of file
+.AppContent {
+  padding: 1em;
+}
+
+/* Product grid styles */
+  .ProductContainer {
+    padding: 0.75em;
+    height: 100%;
+    color: white;
+    background-color: #a75e7d;
+    border-radius: 15px;
+  }
+
+  /* Vertically aligns product image of ProductPreview (in ProductGrid) */
+  .ProductImageHelper {
+    display: inline-block;
+    height: 100%;
+    vertical-align: middle;
+  }
+
+  .ProductImage {
+    vertical-align: middle;
+    max-height: 200px;
+    max-width: 100%;
+  }
+
+  .ProductButton {
+    color: rgb(203, 253, 203);
+    border-color: rgb(78, 219, 78);
+    transition: 0.3s all ease-in-out;
+  }
+
+  .ProductButtonContained {
+    background-color: rgb(72, 136, 72);
+  }
+
+  .ProductButton:hover, .ProductButtonContained:hover {
+    background-color: rgba(79, 255, 108, 0.198);
```

Comment:
Question: How is `rgba` different from `rgb`?

Reply:
The _a_ in `rgba` stands for Alpha, which is used for opacity. On top of adjusting the colors using `rgb`, we can adjust opacity as well, to allow for transparency. I did it here to make the hover color more subtle. It's pretty great!

Reply:
Nice! Sounds like a super useful feature, definitely will be using it in the future :)